analyzers/tests/SonarAnalyzer.UnitTest/TestCases/InitializationVectorShouldBeRandom.cs

```diff
@@ -270,4 +257,25 @@ public class CustomAes : Aes
 
         public override void GenerateKey() => throw new NotImplementedException();
     }
+
+    public class SymmetricalEncryptorWrapper
+    {
+        private readonly SymmetricAlgorithm algorithm;
+
+        public SymmetricalEncryptorWrapper()
+        {
+            algorithm = Aes.Create();
+        }
+
+
+        public void GenerateIV()
+        {
+            algorithm.GenerateIV();
```

Comment:
Same scenario (SV not found) will happen for argument, right? `GenerateIV(SymmetricAlgorithm arg) => arg.GenerateIV();`

Reply:
No. As far as I can tell, for arguments we will get a symbol since they are defined while processing parameter list. The problem appears for class fields.